Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -878,7 +878,42 @@ namespace OpenInfraPlatform {
 					const typename IfcEntityTypesT::IfcTrimmingPreference & trimmingPreference
 				) const throw(...)
 				{
-					throw oip::UnhandledException(clothoid);
+					//throw oip::UnhandledException(clothoid);
+					//check input 
+					if (clothoid.expired())
+						throw oip::ReferenceExpiredException(clothoid);
+					// **************************************************************************************************************************
+					//	https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/schema/ifcgeometryresource/lexical/ifcclothoid.htm

```

Comment:
I'd prefer you take IFC4x3_RC3 as a reference. RC2 has been retired. https://github.com/bSI-InfraRoom/IFC-Documentation

Reply:
Resolved  875f326